.github/workflows/dockerimage.yml

```diff
@@ -17,10 +17,22 @@ jobs:
   build:
     runs-on: ubuntu-22.04
     steps:
-    # Remove pre-populated tools not needed for Docker builds to free up space for cache export
-    # https://github.com/orgs/community/discussions/25678#discussioncomment-5242449
-    - name: Delete tools folder
-      run: rm -rf /opt/hostedtoolcache
+    - name: Maximize build space
+      uses: easimon/maximize-build-space@master
```

Comment:
I'm not familiar with this action, I've used https://github.com/rvolosatovs/nixify/blob/main/.github/workflows/check.yml#L31-L35 myself with good results though for quite a while (just an FYI)

Reply:
Interesting, these two actions do very similar things! I think `easimon` is a bit more advanced because in addition to uninstalling most of the same packages, it also takes the free space from the two available block devices and combines it using LVM to create a single partition with ~100GB of free space on the current generation of runners. I'm using this partition to host Docker's `data-root` which should give us more than enough headroom going forward.